std_msgs/Header header
Area[] areas
DockingStation[] docking_stations
